app/models/concerns/metadata.rb

```diff
@@ -39,6 +39,11 @@ module Metadata
       index.as :displayable
     end
 
+    # For Merritt ARKs
+    property :merritt_id, predicate: RDF::Vocab::DC.identifier do |index|
```

Comment:
@chrissyrissmeyer so we won't be using these ARKs as canonical?

Reply:
We'll be using the Merritt ARKs as canonical for new objects imported from Merritt, but there are existing objects that have both an ADRL ARK, and a merritt ark. this property, or something like it, needs to exist so we can add the Merritt ARKs to legacy objects.

Maybe a rename to `:legacy_id` or `:external_id` or even `legacy_merritt_id` is useful?

Reply:
The `predicate:` needs to change. If it matches the predicate of the `:identifier` property, these accessors will effectively be aliases.

Maybe `RDF::Vocab::DC11.identifier`? or else something custom?

Reply:
@supsrinivas for now *anything* that's different from existing predicates should be suitable. we'll probably want @chrissyrissmeyer's sign off on the predicate before adding values in production.